src/lib.rs

```diff
@@ -298,7 +298,7 @@ fn get_num_cpus() -> usize {
     target_os = "fuchsia")
 )]
 fn get_num_cpus() -> usize {
-    let cpus = unsafe { libc::sysconf(libc::_SC_NPROCESSORS_ONLN) };
+    let cpus = unsafe { libc::sysconf(libc::_SC_NPROCESSORS_CONF) };
```

Comment:
According to the original report, it's a problem on ARM systems that have the `sysconf` function. I've  updated the code to use the constant only when the `target_arch` is ARM.